### PR TITLE
Support protocol relative image URLs by prepending 'https:'

### PR DESCRIPTION
**Pull request by colinta**

### CODE REVIEW

File: Sources/Controllers/Stream/Cells/StreamImageCell.swift [hunk 1]
```diff
@@ -153,6 +153,12 @@ public class StreamImageCell: StreamRegionableCell {
     }
 
     private func loadImage(url: NSURL) {
+        guard url.scheme != "" else {
+            if let urlWithScheme = NSURL(string: "http:\(url.absoluteString)") {
```
Comment (steam) [inline Sources/Controllers/Stream/Cells/StreamImageCell.swift]: I wonder if `http` or `https` is a better default. Probably `http`, @jayzes what are your thoughts?

Comment (colinta) [inline Sources/Controllers/Stream/Cells/StreamImageCell.swift]: Actually, thinking about this again, a protocol-relative URL *implies* that the domain supports both `http` *and* `https`.  I'll switch it up unless @jayzes objects...?

Comment (jayzes) [inline Sources/Controllers/Stream/Cells/StreamImageCell.swift]: HTTPS SGTM

**Comment by colinta** [comment]
Same two weird (unrelated) spec failures that we've seen before.

**Pull request merged**